package-lock.json

```diff
@@ -8638,7 +8638,7 @@
           "dependencies": {
             "minimist": {
               "version": "1.2.0",
-              "resolved": false,
+              "resolved": "",
```

Comment:
This is a strangely tiny change, can you double check your npm and node versions match what this repo asks for in https://github.com/bbc/psammead/blob/latest/.npmrc and https://github.com/bbc/psammead/blob/latest/.nvmrc

Reply:
Also can you run an `npm install` at the base level to update the package-lock.json version please :)